src/plugins/youtubePlayer/plugin.js

```diff
@@ -142,6 +142,10 @@ function setCurrentSrc(instance, elem, options) {
                             } else if (event.data === YT.PlayerState.PAUSED) {
                                 events.trigger(instance, 'pause');
                             }
+                        },
+                        'onError': () => {
+                            instance.destroy();
+                            loading.hide();
```

Comment:
`suicidal plugin` :)

@matjaz321 What if plugin will `reject` promise here?

As for `loading.hide`, I think it should be moved as `.finally(() => loading.hide());` to here https://github.com/jellyfin/jellyfin-web/blob/a50e4853e3206934c1fcaa9a950beb476e3a416f/src/plugins/youtubePlayer/plugin.js#L201

Then this can be removed:
https://github.com/jellyfin/jellyfin-web/blob/a50e4853e3206934c1fcaa9a950beb476e3a416f/src/plugins/youtubePlayer/plugin.js#L117-L119

Reply:
w8 so `instance.destroy()` can reject a promise?

Reply:
> w8 so `instance.destroy()` can reject a promise?

I don't think so. Try replace it with `reject();`. If it won't fix target issue then we should find correct place to destroy player (somewhere in `playbackManager`), but at least Promise will be correctly finished.

Reply:
got it thx

Reply:
Okei I fixed that... Should we show any error message?
So as I mentioned if video is the current UX behavior is not that nice...